schemas/announcements.graphql

```diff
@@ -0,0 +1,42 @@
+### Types
+
+scalar AnnouncementID
+
+enum AnnouncementFields {
+    message
+    importance
+}
```

Comment:
Is this used anywhere?

Reply:
Nope. Leftovers from dumb guide gql adaptation